docs/intro/overview.rst

```diff
@@ -19,143 +19,221 @@ so you can get an idea of how it works and decide if Scrapy is what you need.
 When you're ready to start a project, you can :ref:`start with the tutorial
 <intro-tutorial>`.
 
-Pick a website
+Write a Spider
 ==============
 
 So you need to extract some information from a website, but the website doesn't
-provide any API or mechanism to access that info programmatically.  Scrapy can
-help you extract that information.
+provide any API or mechanism to access that info programmatically. Scrapy can
+help you extract that information. You need to create Scrapy Spider.
+
+A Spider is just an asynchronous web client surrounded by 
+couple of mechanisms that are processing requests and data returned after
+parsing responses. 
+
+Spider gives you following benefits over plain synchronous HTTP library such 
+as urllib or requests, Scrapy Spider:
+
+* handles typical HTTP scenarios, for example manages cookies,
+  redirects, retries on HTTP error codes, filters duplicate
+  requests, limits spider speed, adds headers (such as 'User-Agent');
+* is asynchronous (non-blocking), it can download and process requests 
+  concurrently. This means that Scrapy Spider will always be faster 
+  and more efficient then synchronous client;
+* makes data extraction easier by allowing you to define common
+  data models;
```

Comment:
I'd say it is a wart that you need to define Items upfront, in such a verbose way as in Scrapy :) This makes it harder to export dynamic data (e.g. JSON dicts - we even have a FAQ entry for dynamic items, and it is tricky) and requires more boilerplate. Not sure if it is an adavantage.